sdk/analytics/src/main/java/com/klaviyo/analytics/networking/requests/PushTokenApiRequest.kt

```diff
@@ -78,7 +78,7 @@ internal class PushTokenApiRequest(
 
     override fun equals(other: Any?): Boolean {
         return when (other) {
-            is PushTokenApiRequest -> body.toString() == other.body.toString()
+            is PushTokenApiRequest -> body.toString() == other.body.toString() && query.toString() == other.query.toString()
```

Comment:
since query contains api key

Reply:
@evan-masseau should i also update the hashCode function too?

Reply:
ah yeah totally. actually that makes me wonder if we should just be comparing the hashcodes in this equals function